pkg/image/prune/imagepruner.go

```diff
@@ -491,7 +491,15 @@ func (p *imagePruner) Run(pruneImage ImagePruneFunc, pruneStream ImageStreamPrun
 	pruneImages(p.g, imageNodes, pruneImage, pruneStream, pruneManifest)
 
 	layerNodes := layerNodeSubgraph(allNodes)
-	pruneLayers(p.g, layerNodes, pruneLayer, pruneBlob)
+	registryURL := ""
+	if len(imageNodes) > 0 {
+		if ref, err := imageapi.ParseDockerImageReference(imageNodes[0].Image.DockerImageReference); err == nil {
```

Comment:
> This is legal because we only support a single internal docker registry, right? If so, can you add that comment for when I forget?

yes, will do

> Also, did we ever decided on whether we wanted an endpoint available on the API server that would give a location for hitting the internal docker registry? Feels like that would be useful.

I really think the right way to handle the registry is to create a route with a real DNS entry, if you want to be able to access it from outside of the cluster.